src/main/java/com/example/controller/SimulationController.java

```diff
@@ -374,6 +390,27 @@ public void stop() {
         setSimulationFlagsToNotStartedState();
     }
 
+    public void openDocumentation(ActionEvent actionEvent) throws IOException {
```

Comment:
Couldn't we separate this documentation to another java class? It could be Simulation listener or something of this kind

Reply:
done